tests/playwright/src/model/pages/pods-page.ts

```diff
@@ -80,4 +80,15 @@ export class PodsPage extends MainPage {
     }
     return row.getByRole('button', { name: 'kebab menu', exact: true });
   }
+
+  public async deployedPodExists(podName: string): Promise<boolean> {
```

Comment:
returns true only if pod is deployed to a cluster? I would add a param, what env. is expected for the pod to be in. I am guessing that also some ambiguity is possible when having pod in podman env. and in kubernetes?

Reply:
Since a pod name consists of the pod name itself, the control plane where it is deployed, and the namespace, I don't think there's any reason for ambiguity.
`const DEPLOYED_POD_NAME: string = `${CONTAINER_NAME} ${KIND_CONTAINER_NAME} ${NAMESPACE}`;`